src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/documents/RecallNotificationGotenbergComponentTest.kt

```diff
@@ -30,6 +30,7 @@ class RecallNotificationGotenbergComponentTest : GotenbergComponentTestBase() {
 
 //    writeBase64EncodedStringToFile("recall-notification.pdf", recallNotification.content)
     assertThat(recallNotification, hasNumberOfPages(equalTo(3)))
+    assertThat(recallNotification, hasTotalPageCount(3))
```

Comment:
what's the difference between these 2?

Reply:
total page count is the number that is rendered in the document

Reply:
maybe needs a better name :)